visivo/commands/options.py

```diff
@@ -55,11 +55,11 @@ def dist_dir(function):
     return function
 
 
-def name_filter(function):
+def dag_filter(function):
     click.option(
-        "-nf",
-        "--name-filter",
-        help="Run the command to only include the dag that includes the node with the given name",
+        "-df",
```

Comment:
I wonder if we should make this `--select` or `-sl` rather than dag filter? I think dag filter is more clear. Only reason we'd do select is because it's what dbt uses.

Reply:
Honestly happy to do either, small change.